assets/css/main.css

```diff
@@ -0,0 +1,155 @@
+/** Reset some basic elements */
```

Comment:
this is a copy of the css from the template, with one line changed - the site title css to 75px b/c it needed to be a little bigger. This is how I figured out how to override the template but lmk if better way.